Vienna/Info.plist

```diff
@@ -125,7 +125,7 @@
 		<true/>
 	</dict>
 	<key>NSAppleEventsUsageDescription</key>
-	<string></string>
+	<string>Vienna's blog editor plug-ins use Apple events to share article data. Script plug-ins and AppleScript files may use Apple events as well.</string>
```

Comment:
This causes a warning.

...`/vienna-rss/Vienna/Info.plist:128:16 missing terminating ' character [-Winvalid-pp-token]`